### PR TITLE
Support query browser error log

### PR DESCRIPTION
**Pull request by arugal**
ref: [skywalking#4166](https://github.com/apache/skywalking/issues/4166)

### CODE REVIEW

**Comment by wu-sheng** [comment]
Let's hold this PR until @Fine0830 is back from the vacation.

**Comment by Fine0830** [comment]
Today, I have time to review this PR.

**Comment by arugal** [comment]
> All of `time` fields are able to be generated on server when server get logs from browser? @arugal

When `time == 0`, `time` is set to the current time on the server side.

**Comment by Fine0830** [comment]
> > All of `time` fields are able to be generated on server when server get logs from browser? @arugal
> 
> When `time == 0`, `time` is set to the current time on the server side.

@arugal  If `time` is able to be generated with the current time on server side, we need to remove it from there.

File: browser-error-log.graphqls [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    time: Long!
+}
+
+# Represent the conditions used for query BrowserErrorLogBrief
+input BrowserErrorLogQueryCondition {
+    # The value of 0 means all services.
+    serviceId: ID
+    serviceVersionId: ID
+    pagePathId: ID
+    pagePath: String
```
Comment (Fine0830) [inline browser-error-log.graphqls]: I think we just need one of them.

File: browser-error-log.graphqls [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+type BrowserPerfData {
+    uniqueId: String!
+    serviceId: ID!
+    serviceName: String!
+    serviceVersionId: ID!
+    serviceVersionName: String!
+    pagePathId: ID!
+    pagePath: String!
```
Comment (Fine0830) [inline browser-error-log.graphqls]: As above.

File: browser-error-log.graphqls [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+# Represent the conditions used for query BrowserErrorLogBrief
+input BrowserErrorLogQueryCondition {
+    # The value of 0 means all services.
+    serviceId: ID
+    serviceVersionId: ID
+    pagePathId: ID
+    pagePath: String
+    # The time range of browser perf data time
+    queryDuration: Duration
+    paging: Pagination!
```
Comment (Fine0830) [inline browser-error-log.graphqls]: I don't understand `queryDuration ` and `paging ` fields. Every log time is the current time.

File: browser-error-log.graphqls (hunk 1 above)
Comment (wu-sheng) [reply browser-error-log.graphqls]: I think we could keep pathId only.

File: browser-error-log.graphqls (hunk 3 above)
Comment (wu-sheng) [reply browser-error-log.graphqls]: This is UI query protocol, not client-js upstream protocol.

**Comment by wu-sheng** [comment]
> If time is able to be generated with the current time on server side, we need to remove it from there.

This is the UI/CLI query protocol.

File: browser-error-log.graphqls (hunk 3 above)
Comment (Fine0830) [reply browser-error-log.graphqls]: OK

**Comment by wu-sheng** [comment]
Please follow https://github.com/apache/skywalking/pull/4228 comments and update this PR naming to keep consistent.

**Comment by wu-sheng** [comment]
@Fine0830 Do we provide log query from UI?

**Comment by Fine0830** [comment]
> @Fine0830 Do we provide log query from UI?

Hum... we need to provide error logs query.  I have no ideas for other logs.

**Pull request closed without merging**